src/lib/requests/getRainData.ts

```diff
@@ -0,0 +1,45 @@
+import { getBaseUrl } from '@lib/utils/urlUtil'
+
+type RawRainDataType = {
+  weekday: string
```

Comment:
I know this is still in discussion, but we should remember to update this once the PR for rain API route is finalized.